src/Sulu/Bundle/AudienceTargetingBundle/Resources/public/js/components/target-groups/list/main.js

```diff
@@ -0,0 +1,166 @@
+define([
+    'services/suluaudiencetargeting/target-group-manager',
+    'services/suluaudiencetargeting/target-group-router',
+    'text!./list.html'
+], function(Manager, Router, list) {
+    var defaults = {
+        templates: {
+            list: list
+        },
+
+        translations: {
+            title: 'sulu_audience_targeting.target-groups'
+        }
+    };
+
+    return {
+        defaults: defaults,
+
+        /**
+         * Handles header logic.
+         *
+         * @returns {Object}
+         */
+        header: function() {
+            return {
+                title: this.translations.title,
+                underline: false,
+
+                noBack: true,
+
+                toolbar: {
+                    buttons: {
+                        add: {
+                            options: {
+                                callback: function() {
+                                    Router.toAdd();
+                                }
+                            }
+                        },
+                        deleteSelected: {
+                            options: {
+                                callback: function() {
+                                    this.sandbox.emit(
+                                        'husky.datagrid.target-groups.items.get-selected',
+                                        this.showDeleteConfirmation.bind(this)
+                                    );
+                                }.bind(this)
+                            }
+                        }
+                    }
+                }
+            };
+        },
+
+        layout: {
+            content: {
+                width: 'max'
+            }
+        },
+
+        /**
+         * Constructor of list component.
+         */
+        initialize: function() {
+            this.render();
+
+            this.bindCustomEvents();
+        },
+
+        /**
+         * Renders list component.
+         */
+        render: function() {
+            this.$el.html(this.templates.list());
+
+            this.sandbox.sulu.initListToolbarAndList.call(this,
+                'target-groups',
+                Manager.url() + '/fields',
```

Comment:
Please call this variable `TargetGroupManager`, as in the other places. And I would rename the method to something like `getUrl`.